tests/tunit/tool1394.c

```diff
@@ -27,88 +27,95 @@
 
 #include "memdebug.h" /* LAST include file */
 
+struct cert1394 {
+  const char *param;
+  const char *cert;
+  const char *passwd;
+};
+
```

Comment:
By moving the struct definition within the function scope, the number can be deleted from its name.